pkg/f1/root_cmd.go

```diff
@@ -0,0 +1,73 @@
+package f1
+
+import (
+	"os"
+	"runtime"
+	"runtime/pprof"
+
+	"github.com/form3tech-oss/f1/internal/support/errorh"
+
+	"github.com/form3tech-oss/f1/pkg/f1/fluentd_hook"
+	"github.com/form3tech-oss/f1/pkg/f1/testing"
+
+	"github.com/form3tech-oss/f1/pkg/f1/scenarios"
+
+	"github.com/form3tech-oss/f1/pkg/f1/chart"
+	"github.com/form3tech-oss/f1/pkg/f1/run"
+	"github.com/form3tech-oss/f1/pkg/f1/trigger"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+func buildRootCmd(s *testing.Scenarios, p *profiling) *cobra.Command {
+	rootCmd := &cobra.Command{
+		Use:              "f1",
```

Comment:
probably need to generate this, since each binary will have a different name? Not entirely sure how to do that though!

Reply:
I think it would be best to use `path.Base(os.Args(0))` for that.